.buildkite/pipeline-release.yml

```diff
@@ -35,4 +35,7 @@ steps:
         if: |
           build.message =~ /^buildkite test .*operatorhub.*/
           || build.message =~ /^release eck .*operatorhub.*/
-        command: buildkite-agent pipeline upload .buildkite/pipeline-release-redhat.yml
+        trigger: cloud-on-k8s-operator-redhat-release
```

Comment:
Ok, it took me a minute here to understand that this was triggering another pipeline. Gotcha. Hadn't dealt much with the `trigger` command.